tests/test_fileinfo.py

```diff
@@ -100,10 +102,19 @@ def test_checksums(client):
         assert file_info["checksumType"] == "sha256"
 
     # big file, we should not have a checksum
+    response = requests.get(
+        "https://raw.githubusercontent.com/t-davidson/hate-speech-and-offensive-language/master/data/labeled_data.csv"
```

Comment:
can we use datasets from our own github? we can create a new repo for this if needed

Reply:
I think creating a separate repo for datasets is a great idea as we have for c2d algo.
I'll replace the URL with what we have already.